lib/govuk_publishing_components/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukPublishingComponents
-  VERSION = '12.7.1'.freeze
+  VERSION = '12.8.0'.freeze
```

Comment:
The update of the version usually happens as a separate PR https://github.com/alphagov/govuk_publishing_components/pull/622/files